nuxeo-easyshare-web-ui/src/main/resources/OSGI-INF/nuxeo-easyshare-webresources-contrib.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.easyshare.web.ui.resources.contrib">
+  <require>org.nuxeo.web.ui.resources</require>
+  <extension target="org.nuxeo.ecm.platform.WebResources" point="resources">
+    <resource name="nuxeo-easyshare-share.html" type="import" shrinkable="false">
+      <uri>/ui/nuxeo-easyshare/nuxeo-easyshare-share.html</uri>
+    </resource>
+    <resource name="easysharefolder.svg" type="import" shrinkable="false">
+      <uri>/ui/images/doctypes/easysharefolder.svg</uri>
+    </resource>
```

Comment:
No need to import this as a resource, because it is already be copied to the respective folder and will be loaded by convention.